BetterTeams/src/main/java/com/booksaw/betterTeams/integrations/placeholder/provider/AnchorPlaceholderProvider.java

```diff
@@ -0,0 +1,12 @@
+package com.booksaw.betterTeams.integrations.placeholder.provider;
```

Comment:
Could you add the new placeholders to the wiki as well: https://github.com/booksaw/BetterTeams/blob/master/wiki/docs/05-dependencies/PlaceholderAPI.md 

It massively reduces the amount of queries we get in the discord if the wiki is kept up to date :)